openwisp-monitoring/files/sbin/netjson-monitoring.lua

```diff
@@ -164,6 +164,12 @@ for name, interface in pairs(network_status) do
       netjson_interface.statistics = interface.statistics
     end
     local addresses = monitoring.interfaces.get_addresses(name)
+    local virtual_interfaces = {'wireguard'}
+
+    if next(addresses) and
+      monitoring.utils.has_value(virtual_interfaces, addresses[1].proto) then
```

Comment:
I don't think I need to check all addresses. Only one should be enough.

Correct me if I am wrong 🙃